### PR TITLE
use globalThis instead of process.env to set env at runtime

### PR DESCRIPTION
**Pull request by aruniverse**
configure env at runtime instead of only at buildtime

see related pr: https://github.com/imodeljs/create-react-app/pull/70

### CODE REVIEW

**Review by johnnyd710 — APPROVED** [review]
nice so we don't have to rebuild if we forget to set the dev environment? :)

File: packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx [hunk 1]
```diff
@@ -142,10 +142,10 @@ const ViewerHome: React.FC = () => {
           new MeasureToolsUiItemsProvider(),
         ]}
         extensions={[
-          new LocalExtensionProvider({
-            manifestPromise: LocalExtension.manifestPromise,
-            main: LocalExtension.main,
-          }),
+          // new LocalExtensionProvider({
```
Comment (kckst8) [inline packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx:145]: why is this commented?

File: packages/apps/web-viewer-test/tsconfig.json [hunk 2]
```diff
@@ -13,8 +17,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
```
Comment (kckst8) [inline packages/apps/web-viewer-test/tsconfig.json:20]: isn't react-jsx the new value (React 17)?

Comment (aruniverse) [reply packages/apps/web-viewer-test/tsconfig.json:20]: this is one of those changes made by cra, ill revert

Comment (aruniverse) [reply packages/apps/web-viewer-test/tsconfig.json:20]: image

actually looks like this is the right one we want for react 17

File: packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx (hunk 1 above)
Comment (aruniverse) [reply packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx]: had run into this earlier, had to clean and rebuild a few times before it worked. 
reverted.

image

**Review by kckst8 — APPROVED** [review]

**Pull request merged**